.github/workflows/release-docker-image.yaml

```diff
@@ -0,0 +1,56 @@
+name: Release Self Serve Portal Image
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} Self Serve Portal Image by @${{ github.actor }}  
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: 'The type of release'
```

Comment:
No need for quotes, and do we need `required: true`?

Reply:
Because it's a type: choice, I don't think we need to add required: true as it's a dropdown menu